api/models/record_model.py

```diff
@@ -19,7 +19,7 @@ def __init__(
         self.record_title = record_title
         self.record_geolocation = record_geolocation
         self.record_type = record_type
-        self.status = 'Pending'
+        self.status = 'Under Investigation'
         self.record_placement_date = datetime.datetime.now().strftime\
```

Comment:
I think this method call is redundant because in your table you have catered to this value already

Reply:
`okay got it let me work on it.